.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Very good by adding `test.md` file, you can also add `node_modules`, so if you have any packages don't get uploaded also.